src/v3/test/integration/__snapshots__/email-challenge-consent.test.tsx.snap

```diff
@@ -151,42 +151,29 @@ exports[`email-challenge-consent should render form 1`] = `
                           data-se="icon-browser"
                         >
                           <svg
-                            aria-hidden="true"
```

Comment:
Is there any concern that these will no longer use the aria-hidden attr?

Reply:
Good catch!

Yes, there is a concern. Hiding icon title from accessibility seems desirable behaviour.

However Odyssey uses MUI's  [SvgIcon](https://mui.com/material-ui/api/svg-icon/#SvgIcon-prop-titleAccess) which sets `aria-hidden="true"` only if `titleAccess` is not passed:
https://github.com/mui/material-ui/blob/a656b90a3b616a268c7a8c6ddc88e7d536263b25/packages/mui-material/src/SvgIcon/SvgIcon.js#L107

If we [pass `titleAccess`](https://github.com/okta/okta-signin-widget/pull/3476/files#diff-794b3e864714ca56eafbdcb9160fc3c3f734f51e74f2bf1e264eec34f11aae29R26), MUI adds `<title>` with icon description inside `<svg>` (in UI it will present tooltip when we hover on icon, also description will be read by speech synthesizers like VoiceOver)
If we don't pass `titleAccess`, there will be no tooltip. 

@alexdahl-okta @lesterchoi-okta

Reply:
Fixed